crates/hyperqueue/src/client/output/cli.rs

```diff
@@ -395,10 +395,20 @@ impl Output for CliOutput {
     }
 
     fn print_job_submitted(&self, job: JobDetail) {
+        let resources = match job.job_desc {
```

Comment:
I would remove this change. Graph jobs are not submitted using the CLI and for normal submits it's not really helpful, since it was specified in the command arguments.

Reply:
What was the removed change?

Reply:
Printing the resource requests immediately after submit.